### ISSUE TITLE
Data Availability

### ISSUE THREAD

**Issue by Yijia-Xiao** [pwr-inf/hex2vec#1]
Hi,

I am trying to use the repo to carry out some downstream tasks. However, when I run the `make_dataset` script, after downloading the required data (~93G in data/, ~17G in cache/), the processing part failed. Could you provide possible solutions?

Thank you!

--
Best,
Yijia
Below is the error information
```
Tag: waterway exists for city: Philadelphia, USA
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 29/29 [00:00<00:00, 193.89it/s]
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 54/54 [04:54<00:00,  5.46s/it]
Downloading finished
  0%|                                                                                                                                         | 0/54 [00:00<?, ?it/s]Tag aerialway doesn't exist for city Tirana, Albania, skipping...
  0%|                                                                                                                                         | 0/54 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "PATH_TO/hex2vec/make_dataset.py", line 114, in <module>
    add_h3_indices_to_city(city, 8)
  File "PATH_TO/hex2vec/src/data/make_dataset.py", line 111, in add_h3_indices_to_city
    tag_gdf = tag_gdf[['unique_id', tag, 'geometry']]
  File "PATH_TO/anaconda3/envs/geovex/lib/python3.10/site-packages/geopandas/geodataframe.py", line 1428, in __getitem__
    result = super().__getitem__(key)
  File "PATH_TO/anaconda3/envs/geovex/lib/python3.10/site-packages/pandas/core/frame.py", line 3811, in __getitem__
    indexer = self.columns._get_indexer_strict(key, "columns")[1]
  File "PATH_TO/anaconda3/envs/geovex/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6108, in _get_indexer_strict
    self._raise_if_missing(keyarr, indexer, axis_name)
  File "PATH_TO/anaconda3/envs/geovex/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 6171, in _raise_if_missing
    raise KeyError(f"{not_found} not in index")
KeyError: "['unique_id'] not in index"
```

**Comment by mschrader15** [comment]
Hi @Yijia-Xiao, we ran into the same issues and have fixed it in #2.

**Comment by Yijia-Xiao** [comment]
Hi @mschrader15,

Got it. Thank you!
